### ISSUE TITLE
Laravel 5.2 Error

### ISSUE THREAD

**Issue by stueynet** [Grimthorr/laravel-user-settings#9]
This is new.

```
'ErrorException' with message 'json_decode() expects parameter 1 to be string, array given' in /home/vagrant/Code/maple/vendor/grimthorr/laravel-user-settings/src/Setting.php:227
```

**Comment by balping** [comment]
Here's the solution. Apply my patch if you don't want to wait until it's merged

https://github.com/Grimthorr/laravel-user-settings/pull/10

**Comment by Grimthorr** [comment]
Thanks both - I've merged this into version 2.1.0. For Laravel versions <5.2, support is still available using [version 2.0.x](https://github.com/Grimthorr/laravel-user-settings/tree/laravel5).
